app/Rules/NotExistsRule.php

```diff
@@ -37,7 +37,7 @@ public function passes($attribute, $value)
             : $this->parameters[1];
 
         return DB::table($tableName)
-            ->where($columnName, $value)
+            ->where($columnName, 'like', $value)
```

Comment:
Do you need to surround the $value with %%

Reply:
it's a cheat, i'm doing it to deal with uppercase/lowercase things.

Reply:
i'll comment that.

Reply:
scratch that, changed it.